package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "messagebird",
-  "version": "2.2.0",
+  "version": "2.2.1",
```

Comment:
I think following semver this should be a `minor` instead because we added new features 🎉

Reply:
Also, maybe do version bumps in separate commits outside of a PR? You might want to include multiple PRs before releasing a version